easybuild/easyconfigs/t/TRUST/TRUST-3.0.2-intel-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = "PythonPackage"
+
+name = 'TRUST'
+version = '3.0.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://bitbucket.org/liulab/trust'
+description = """ Tcr Receptor Utilities for Solid Tissue (TRUST) is a computational tool to analyze
+ TCR and BCR sequences using unselected RNA sequencing data, profiled from solid tissues, including tumors. """
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['https://bitbucket.org/liulab/%(namelower)s/get']
+sources = ['%(version)s.tar.gz']
+checksums = ['05144733b9296298ef21d7fd7aecd72e19da7cdbd8323ac5b9c0782a1b407da0']
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('Pysam', '0.14.1', versionsuffix),
+    ('Biopython', '1.71', versionsuffix),
+    ('python-parasail', '1.1.12', versionsuffix),
+]
+
+sanity_check_paths = {
+    'files': ['bin/f2py', 'bin/trust'],
+    'dirs': ["lib/python%(pyshortver)s/site-packages/%(namelower)s-%(version)s-py%(pyshortver)s-linux-x86_64.egg"],
```

Comment:
`f2py` is installed through a side-effect, not directly via `TRUST`, so I wouldn't check for that

Also don't check for the actuall `.egg` directory (since it could be a file too, depending on circumstances), just check for `lib/python%(pyshortver)s/site-packages`.